### PR TITLE
Adjust the content a bit

### PR DESCRIPTION
**Pull request by kamilkisiela**

### CODE REVIEW

File: packages/web/docs/src/components/landing-page.tsx [hunk 1]
```diff
@@ -154,7 +153,7 @@ function EnterpriseFocusedCards({ className }: { className?: string }) {
       className={cn('bg-beige-100 rounded-3xl px-4 pt-6 sm:py-24 md:px-6 md:py-[120px]', className)}
     >
       <Heading as="h3" size="md" className="text-balance sm:px-6 sm:text-center">
-        Enterprise-focused tooling at your disposal
+        Enterprise-focused tools tailored for you.
```
Comment (hasparus) [inline packages/web/docs/src/components/landing-page.tsx:156]: Just FYI, one card has a full stop `.`, the next one exclamation mark `!`, and the text for Single Sign-On just ends without any interpunction.

File: packages/web/docs/src/components/ecosystem-management.tsx [hunk 2]
```diff
@@ -132,7 +132,7 @@ function Illustration(props: { className?: string }) {
         </Edge>
         <div>
           <Node
-            title="Mesh"
+            title="Hive"
```
Comment (hasparus) [inline packages/web/docs/src/components/ecosystem-management.tsx:135]: I kinda feel like two tiles with the title `Hive` will seem like bug to the reader.

```suggestion
            title="Hive Gateway"
```

Would this make sense too?

**Review by ardatan — APPROVED** [review]

File: packages/web/docs/src/components/landing-page.tsx (hunk 1 above)
Comment (kamilkisiela) [reply packages/web/docs/src/components/landing-page.tsx:156]: good point, I will add `?` to Single Sign-On to follow the rule :D

Comment (hasparus) [reply packages/web/docs/src/components/landing-page.tsx]: exactly what I envisioned <3

**Review by saihaj — APPROVED** [review]

**Review by dotansimha — APPROVED** [review]

**Review by hasparus — APPROVED** [review]

**Review by n1ru4l — APPROVED** [review]

File: packages/web/docs/public/graphql-logo.svg
Comment (n1ru4l) [inline packages/web/docs/public/graphql-logo.svg:1]: I don't think its okay to present the GraphQL logo like this

**Pull request merged**